storage.bs

```diff
@@ -283,25 +297,13 @@ steps:
   <ol>
    <li><p>Let <var>origin</var> be <var>settingsObject</var>'s <a>origin</a>.
 
-   <li><p>Let <var>permission</var> be <a>permission</a> for <var>origin</var>.
-
-   <li>
-    <p>If <var>permission</var> is "<code title>prompt</code>", then determine through asking the
-    user or using heuristics whether <var>origin</var> is allowed to use persistent storage. If that
-    returns a positive answer, then set <var>permission</var> to "<code title>granted</code>". If
-    that returns a negative answer, then set <var>permission</var> to "<code title>denied</code>".
-    If that returns no answer, then do not alter <var>permission</var>.
-
-    <p class="note">User agents are encouraged to not let the user answer this question twice for
-    the same <a>origin</a> around the same time and this algorithm is not equipped to handle such a
-    scenario.
-
-   <li><p>Set <a>permission</a> for <var>origin</var> to <var>permission</var>.
+   <li><p><a>Request permission to use</a> <code>{name: {{"persistent-storage"}}}</code>,
+   and let |permission| be the result.
```

Comment:
I would prefer to lead with "Let _permission_ be ...".

Also, is this descriptor syntax defined and explained somewhere?